fe/fe-core/src/main/java/com/starrocks/sql/analyzer/QueryAnalyzer.java

```diff
@@ -312,11 +312,10 @@ public Scope visitTable(TableRelation node, Scope outerScope) {
                 Field field;
                 if (baseSchema.contains(column)) {
                     field = new Field(column.getName(), column.getType(), tableName,
-                            new SlotRef(tableName, column.getName(), column.getName()), true);
+                            new SlotRef(tableName, column.getName(), column.getName()), true, column.isAllowNull());
```

Comment:
I think we could directly set the nullable property to SlotRef

Reply:
the expr is resolved from bottom to up. the path is: resolve table -> resolve SlotRef. After resolving table, we got scope and related fields. when resolving SlotRef, we get nullable from fields. so I should keep nullable info here